plone-5.1.x.cfg

```diff
@@ -15,10 +15,12 @@ plone.schema = 1.2.1
 # Last pyrsistent version that is python 2 compatible:
 pyrsistent = 0.15.7
 
+# codeanalisys
+typing = 3.7.4
```

Comment:
@sneridagh I think the correct thing here is to pin `importlib-metadata = 2.1.1`. `importlib-metadata >= 3.0.0` is not compatible with Python 2:

https://github.com/python/importlib_metadata/blob/530f5da8dd3a255f1198f29ea0126b8b25e644d8/setup.cfg#L14

Reply:
thanks! I'll try, this build is killing me